### PR TITLE
[BugFix] Disable txngraph in nonnewpublish

### PR DESCRIPTION
**Pull request by decster**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Problem Summary(Required) ：

This is temporary patch to fix the testing environment failure of load image&wal in FE.  For some reason, 2 txn of the same table, their visible order are not same as commit order, which needs further investigation, 

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/transaction/DatabaseTransactionMgr.java|1|1|100.00%|[]|

**Comment by decster** [comment]
run starrocks_admit_test

**Review by chaoyli — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java [hunk 1]
```diff
@@ -1176,7 +1176,7 @@ protected void unprotectUpsertTransactionState(TransactionState transactionState
                     runningTxnNums++;
                 }
             }
-            if (transactionState.getTransactionStatus() == TransactionStatus.COMMITTED) {
+            if (Config.enable_new_publish_mechanism && transactionState.getTransactionStatus() == TransactionStatus.COMMITTED) {
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1179]: Should transactionGraph.remove dependent on this config?

Comment (decster) [reply fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1179]: if it's empty, it wont have any effects

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**